examples/sushi/config.py

```diff
@@ -28,12 +28,28 @@
 )
 
 # Due to a 3.7 mypy bug we ignore. Can remove once 3.7 support is dropped.
-airflow_config = Config(default_scheduler=AirflowSchedulerConfig())  # type: ignore
+airflow_config = Config(  # type: ignore
+    default_scheduler=AirflowSchedulerConfig(),
+    gateways={
+        "spark": {
+            "connection": {
+                "type": "spark",
```

Comment:
This config is used when pytest runs on the host (as opposed to container). Shouldn't we provide a path to the config folder here?